doc/dev/main.xml

```diff
@@ -36,8 +35,7 @@ among the developers before they are committed, called <E>proposals</E>;
 there is a concept of <E>modules</E> where  parts of the
 &GAP; system are assigned to specified maintainers.
 </Abstract>
-
-
+    <Copyright> 2005-2015, The &GAP; Group</Copyright>
```

Comment:
Should we maybe also change that to 2021?

Reply:
I have changed it such that it  is similar to the other manuals (using a `&RELEASEYEAR;` entity).